vcpkg/commands/new.md

```diff
@@ -0,0 +1,136 @@
+---
+title: vcpkg new
+description: Command line reference for the vcpkg new command. Creates a new manifest in the current directory.
+ms.date: 07/05/2023
+---
+# vcpkg new
+
+## Synopsis
+
+```console
+vcpkg new --application
+```
+
+```console
+vcpkg new --name hello --version 1.0
+```
+
+## Description
+
+Writes a new manifest into the current working directory. In `--application` mode, the manifest
+contains no name or version, suitable for use in an end user application that ships in binary form.

```

Comment:
```suggestion
Writes a new manifest into the current working directory. 

```

Reply:
After some consideration, I believe we should rewrite this to reflect that a default configuration is also getting created.

```suggestion
Creates a manifest file (`vcpkg.json`) and default configuration file (`vcpkg-configuration.json`) ready to use in your project. Use the `--application` flag if your project is an end-user application.
```